docs/recipes/gulp/gulpfile.js

```diff
@@ -9,18 +9,16 @@
 const gulp = require('gulp');
 const connect = require('gulp-connect');
 const lighthouse = require('lighthouse');
-const ChromeLauncher = require('lighthouse/lighthouse-cli/chrome-launcher').ChromeLauncher;
+const chromeLauncher = require('lighthouse/chrome-launcher');
```

Comment:
Github is being really weird. Here's what I see:

<img width="998" alt="screen shot 2017-06-19 at 2 29 41 pm" src="https://user-images.githubusercontent.com/238208/27306758-c0b06978-54fb-11e7-82c5-436969c13f7b.png">

Think I have an extension adding to the paths.